tests/src/FunctionalJavascript/MembershipSubmissionTest.php

```diff
@@ -133,6 +133,31 @@ public function testSubmitWebform() {
     $this->getSession()->getPage()->pressButton('Save Settings');
     $this->assertSession()->pageTextContains('Saved CiviCRM settings');
 
+    // Create two memberships with the same status with the first membership
+    // having an end date after the second membership's end date.
+    $this->utils->wf_civicrm_api('membership', 'create', [
+      'membership_type_id' => 'Basic',
+      'contact_id' => 2,
+      'join_date' => '08/10/21',
+      'start_date' => '08/10/21',
+      'end_date' => '08/10/22',
+      'is_override' => 1,
+      'status_id' => 'Expired',
+    ]);
+
+    $this->utils->wf_civicrm_api('membership', 'create', [
+      'membership_type_id' => 'Basic',
+      'contact_id' => 2,
+      'join_date' => '01/01/21',
+      'start_date' => '01/01/21',
+      'end_date' => '01/01/22',
+      'is_override' => 1,
+      'status_id' => 'Expired',
+    ]);
+
+    $this->drupalGet($this->webform->toUrl('canonical'));
+    $this->assertSession()->statusMessageContains('Basic membership for ' . mb_strtolower($this->adminUser->getEmail()) . ' has a status of "Expired". Expired ' . \CRM_Utils_Date::customFormat('2022-08-10'));
```

Comment:
Test fail seems to be related to this function `statusMessageContains()`? It was introduced in [9.4.x version](https://www.drupal.org/node/3270424). @KarinG Do we still need 9.3.x in the test matrix?

Reply:
Yes - we do still need 9.3.x - it's a supported Drupal version so sites are using it. 9.3.21 was released Aug 03 (together with 9.4.5).